utils/errors/errors.go

```diff
@@ -30,8 +30,8 @@ var (
 	// ErrCouldNotParse is returned when a fetcher fails to parse the update data.
 	ErrCouldNotParse = errors.New("updater/fetchers: could not parse")
 
-	//ErrConfigNotLoaded is returned when the configuration file is not loaded properly
-	ErrConfigNotLoaded = errors.New("could not load configuration properly")
+	// ErrDatasourceNotLoaded is returned when the datasource variable in the configuration file is not loaded properly
+	ErrDatasourceNotLoaded = errors.New("could not load datasource configuration properly")
```

Comment:
We try to declare the errors are locally as possible - it should be in config/config.go as this error is very specific and not meant to be re-used.

Also, I am not sure that everybody will understand that error message. Maybe something like `could not load configuration: no database source specified` would be more appropriate?